superset/assets/package.json

```diff
@@ -24,8 +23,7 @@
   },
   "jest": {
     "coverageReporters": [
-      "text",
-      "cobertura"
+      "text"
     ]
```

Comment:
I think we don't need "jest" section as we are using defaults

Reply:
Removed these steps.